docs/dub-reference/platform_specifications.md

```diff
@@ -33,5 +33,31 @@ For example, to only enable a linker flag `-ld_classic` when on Mac OSX and arch
 
 > Notice that in the JSON format, the platform specification follows a single dash (`-`) after the setting name.
 
-For the full set of platform systems and architectures, see the [D language reference](http://dlang.org/version.html#PredefinedVersions).
+For the full set of platform systems, see the [D language reference](http://dlang.org/version.html#PredefinedVersions).
 Notice that the version names should be converted to lower case. Hence, instead of `iOS`, use `ios`, for example.
+
+## CPU / Target Architectures
+
+The following architectures are supported by DUB, e.g. for the `--arch` flag and are translated to the following underlying command line arguments:
+
+|  Architecture  |     [DMD]     |        [LDC]       |    [GDC]    |
+| -------------- | ------------- | ------------------ | ----------- |
+| `x86_64`       | `-m64`        | `-march=x86-64`    | `-m64`      |
```

Comment:
Is unsupported only referring to dub? All listed are supported by gdc.

The x86/x86_64 example seems strange, as `-m64` may not be recognised by gdc, and in some cases does something different (targets powerpc64 when gdc was built for powerpc and supports multilib)

Reply:
Ok, so it's just x86-centric, or never considered that it might actually run on power8 hardware.